### DISCUSSION TITLE
Is there a list of recommended video cards?

### DISCUSSION THREAD

**Discussion by simondsmason** [blakeblackshear/frigate#8990] [Ask A Question]
I see plenty of reddit comments where people recommend one card.  And in many cases if you google that card you will find people having issues with it.  I am moving to unraid on a Dell R730.  Would like to buy a card and know it is going to work.  So looking for pointers.  Thanks.

**Comment by NickM-27** [comment]
there are a lot more variables than that. Like what all are you wanting to use the card for (video decoding, object detection)? Any power or size constraints?

**Comment by simondsmason** [comment]
Thanks - good questions.  I have a Coral which I think? takes care of detection.  I have 8 cameras.  With 24 cores I believe I will be able to dedicate some of these to help out.  So I am not entirely clear what I can depend on the Xeon processors for vs. the GPU.  Still learning.  In the Dell R730 it states that I can fit up to two double wide video cards but I think one would suffice.  And I plan to use if it for Emby video but that is a secondary need.  Making Frigate happy is my first requirement.

**Reply by NickM-27** [reply]
Generally unraid works best with amd or nvidia. Nvidia will provide the most possibilities like AI (inside frigate or maybe other like face recognition).

**Reply by simondsmason** [reply]
A friend recommended an Nvidia GTX 1060 6GB so I am going to look into one of those.

**Reply by HIT2022** [reply]
> A friend recommended an Nvidia GTX 1060 6GB so I am going to look into one of those.

Hey Simon. How'd that work out for you? I'm looking to migrate to a similar setup (8+ 4k cameras, R730) since I have a 1060 collecting dust. Any stats re: inference speed, CPU use, etc. etc. would be appreciated. I've ran Frigate on NUC8i5BEH + Coral for the last several years, but with the AI features being introduced w/ v16, I suspect that will no longer cut it.

**Reply by simondsmason** [reply]
It seems to work well.  CPU loads are low and speeds are good.  To be honest I didn't do much of a before and after but just went ahead and did it.  I am running 7 cameras - down from 10.  But no big hits on CPU and response times are all good.

Here is a snapshot of what I see in the metrics but happy to provide more stats as needed?

<img width="1470" alt="image" src="https://github.com/user-attachments/assets/4d2d7d90-50a3-4957-884b-977d97126207" />

**Reply by HIT2022** [reply]
@simondsmason 

What's your general CPU usage like? Assuming you're running more than just frigate, but just curious how close to idle your chassis is w/ 7 cameras 24/7'd

**Reply by simondsmason** [reply]
Pretty stable:

<img width="594" alt="image" src="https://github.com/user-attachments/assets/0f4d43d4-d9e8-43db-a65f-b45c3be53760" />

Running 9 cameras right now.  I do run a bunch of other things but they are mostly idle except when someone is watching a movie, etc.  I don’t run any graphing tools as I don’t have the need. I do see the CPU go up when detections occur but not by much.

**Reply by MYeager1967** [reply]
Did that card increase the fan speeds? I'm building out a R730XD now and Frigate/Emby will be the primary workload. 2x Xeon 2695's shouldn't break much of a sweat.

Also, are you still using the Coral or has the GPU taken over those duties?
